staging/src/k8s.io/api/extensions/v1beta1/types.go

```diff
@@ -1001,7 +1001,7 @@ type HostPortRange struct {
 	Max int32 `json:"max" protobuf:"varint,2,opt,name=max"`
 }
 
-// SELinux  Strategy Options defines the strategy type and any options used to create the strategy.
+// SELinux Strategy Options defines the strategy type and any options used to create the strategy.
```

Comment:
if you're going to change this, make them actually match godoc conventions and start with the type name

Reply:
Shall I also update other PSP-related fields to follow this conventions?